torch/csrc/jit/codegen/cuda/lower_scalar_hoist.cpp

```diff
@@ -189,8 +189,7 @@ std::pair<Val*, bool> CommonScalarMap::hoistScalarImpl(
 
 namespace {
 
-std::list<ValInfo> getVariablesFromLoops(
-    const std::vector<kir::ForLoop*>& loops) {
+std::list<ValInfo> getLoopIndices(const std::vector<kir::ForLoop*>& loops) {
```

Comment:
@naoyam Your review https://github.com/csarofeen/pytorch/pull/2215#discussion_r1047827902 is here